js/src/main/scala/org/scalamock/clazz/MockFunctionFinderImpl.scala

```diff
@@ -29,66 +29,23 @@ object MockFunctionFinderImpl {
   def mockedFunctionGetter[M: c.WeakTypeTag](c: Context)
                                             (obj: c.Tree, name: c.Name, targs: List[c.Type], actuals: List[c.universe.Type]): c.Expr[M] = {
     import c.universe._
-    val utils = new MacroUtils[c.type](c)
-    import utils._
-
-    def hasValueTypeArgs(baseSymbol: Symbol, owner: Type): Boolean = {
-      val baseType = owner.baseType(baseSymbol)
-      baseType.typeArgs.nonEmpty // && baseType.typeArgs.forall(_ <:< typeOf[AnyVal])
-    }
-
-    // this somehow replicates postfix logic from scala-js, there have to be a better way
-    def privateSuffix(owner: Tree): String = {
```

Comment:
this has been contentious in the past, I guess newer versions of scala.js improve the toString of the `Name` type enough to make it unnecessary?
Does that introduce a dependency ont hat mechanism later on where the name gets assembled?

Reply:
With my approach I'm not depending on the name generation algorithm in Scala.js - by using `@JSExport(<encoded mock function name>)` annotation in `MockMaker` this decision is kept entirely in Scalamock. If you change `MockMakerInner#mockFunctionName`, you'll need to update this - but it's the same in the JVM impl.

Reply:
thanks!